test/docker-compose-with-error.sh

```diff
@@ -1,60 +1,38 @@
 #!/bin/bash
 
 set -e
-trap compose_stop EXIT
-
-compose_stop() {
-    docker-compose ${FILE_ARGS} --project-name "$p" stop
-}
 
 usage() { 
-    echo 'Usage: $0 -p <project-name> -f docker-compose1.yml [SERVICES]' 1>&2
+    echo 'Usage: $0 [SERVICES]' 1>&2
     echo
-    echo 'This script calls into `docker-compose up` with the supplied arguments' 1>&2
-    echo 'and checks the exit code of each container upon exit. If any container' 1>&2
-    echo 'exit code is non-zero, this script will exit non-zero.' 1>&2
+    echo 'This script calls into `docker-compose up` and checks the exit code' 1>&2
+    echo 'of each container upon exit. If any container exit code is non-zero,' 1>&2
+    echo 'this script will exit non-zero.' 1>&2
     echo
-    echo 'Options:' 1>&2
-    echo '    -f	Path to compose file. Can be passed multiple times.' 1>&2
-    echo '    -p	Project name.' 1>&2
-    echo '    -t	Target (if you only want to execute one thing).' 1>&2
+    echo 'Use Compose environment variables, such as COMPOSE_FILE and COMPOSE_PROJECT_NAME,' 1>&2
+    echo 'for driecting docker-compose.' 1>&2
     exit 1
 }
 
 while getopts "hf:p:t:" arg; do
```

Comment:
You don't need the config flags for `f`, `t`, or `p` anymore.

I'm not actually certain we need this `getopts` bit at all anymore.

Reply:
Agreed, I'll drop it.